tests/byzantium/eip196_ec_add_mul/conftest.py

```diff
@@ -0,0 +1,35 @@
+"""Shared pytest definitions local to EIP-2537 tests."""
+
+import pytest
+from execution_testing import (
+    Fork,
```

Comment:
Required for updated typehint.
```suggestion
    Address,
    Fork,
```